### PR TITLE
Fix NUMA test, improve isolation

### LINKED ISSUE #856: NUMAMemoryResourceTest.AreEqual fails CI on master
https://ares.epic.hpi.uni-potsdam.de/jenkins/blue/organizations/jenkins/Hyrise%2Fhyrise/detail/master/265/pipeline

@mrks In #849 you changed something here?

### PR DESCRIPTION
**Pull request by mrks**
This kind of... fixes #856

### CODE REVIEW

File: src/test/statistics/statistics_import_export_test.cpp [hunk 1]
```diff
@@ -11,7 +12,9 @@
 
 namespace opossum {
 
-class StatisticsImportExportTest : public ::testing::Test {};
+class StatisticsImportExportTest : public ::testing::Test {
+  void TearDown() { std::remove((test_data_path + "exported_table_statistics_test.json").c_str()); }
```
Comment (mrzzzrm) [inline src/test/statistics/statistics_import_export_test.cpp]: y is this necessary? I thought test_data_path makes sure there are no conflicts with filenames and such?

Comment (mrks) [reply src/test/statistics/statistics_import_export_test.cpp]: 1) `test_data_path` is not necessarily set. If you call `./build/hyriseTest` on the current master, you will end up with the json file in your project root.
2) It isolates only different builds (mainly for CI). It does not clear that folder.
3) In any case, each test should clean up after itself.

Comment (mrzzzrm) [reply src/test/statistics/statistics_import_export_test.cpp]: woah, so many reasons

**Comment by mrks** [comment]
Apparently, on an actual NUMA system, the migration indirectly uses `is_equal`. I put it back in for now, because it was not broken before.

**Pull request merged**